bucket/capframex.json

```diff
@@ -6,9 +6,13 @@
     "suggest": {
         "OCAT": "ocat"
     },
-    "url": "https://github.com/CXWorld/CapFrameX/releases/download/v1.6.6/CapFrameX_v1.6.6_Portable.zip",
-    "hash": "f889bd33d52c05172f79d18579cc2249b3e1c750a6ec4daef735e840f637608b",
-    "extract_dir": "CapFrameX 1.6.6 Portable",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/CXWorld/CapFrameX/releases/download/v1.6.6/CapFrameX_v1.6.6_Portable.zip",
+            "hash": "f889bd33d52c05172f79d18579cc2249b3e1c750a6ec4daef735e840f637608b",
+            "extract_dir": "CapframeX 1.6.6 Portable"
```

Comment:
Needs to be `CapFrameX 1.6.6 Portable`; missed the capital `F`.

Reply:
Thanks!!!!